fastlane/Fastfile

```diff
@@ -37,7 +37,16 @@ platform :ios do
       device: "iPhone 8",
       code_coverage: true,
       skip_testing: [
-        "DataTests",
```

Comment:
All Core Data tests fail occasionally, disabling them all to run `SyncCryptoTests` from `Data` framework